src/test/java/seedu/address/model/person/ItemTest.java

```diff
@@ -0,0 +1,38 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class ItemTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Item(null));
+    }
+
+    @Test
+    public void constructor_invalidItem_throwsIllegalArgumentException() {
+        String invalidItem = "";
+        assertThrows(IllegalArgumentException.class, () -> new Item(invalidItem));
+    }
+
+    @Test
+    public void isValidItem() {
```

Comment:
Quite a comprehensive enforcement of constraints here as well. Good use of assertions in test.